CHANGELOG.md

```diff
@@ -10,10 +10,12 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Count IPv4 and IPv6 statistics individually to avoid confusion ([#50])
```

Comment:
```suggestion
- BREAKING: Count IPv4 and IPv6 statistics individually to avoid confusion.
  Because of this, the Prometheus metrics `breakwater_ips` and `breakwater_legacy_ips` have been removed,
  use `breakwater_ips_v4` and `breakwater_ips_v6` instead ([#50])
```